### PR TITLE
MacOS 10.14 Software updates Week 6

### PR DESCRIPTION
**Pull request by al-cheb**
Software updates:
- System Version: macOS 10.14.6 (18G3020)
- Rust 1.41.0
- PowerShell 6.2.4
- Go 1.13.7
- Gradle 6.1.1
- Fastlane 2.141.0
- Cmake 3.16.3
- Microsoft Edge 79.0.309.71 
- MSEdgeDriver 79.0.309.71
- Visual Studio for Mac 8.4.3.12
- Android SDK Build-Tools, Revision 29.0.3

### CODE REVIEW

File: images/macos/macos-10.14-Readme.md [hunk 1]
```diff
@@ -366,3 +360,5 @@ The following software is installed on machines in the Azure Pipelines **macOS-1
 | Google Play services                            | 49                                              |
 | Google Repository                               | 58                                              |
 | Intel x86 Emulator Accelerator (HAXM installer) | 7.5.1                                           |
+
+
```
Comment (sergey-akhalkov) [inline images/macos/macos-10.14-Readme.md]: nitpick: empty lines is a noise, would be great to remove them.

File: images/macos/macos-10.14-Readme.md [hunk 2]
```diff
@@ -4,34 +4,26 @@ The following software is installed on machines in the Azure Pipelines **macOS-1
...
 - Python 2.7.17
 - Python 3.7.6
 - Ruby 2.6.5p114
-- Rust 1.40.0
 - .NET SDK 1.0.1 1.0.4 1.1.4 1.1.5 1.1.7 1.1.8 1.1.9 1.1.10 1.1.11 1.1.12 1.1.13 2.0.0 2.0.3 2.1.2 2.1.4 2.1.100 2.1.101 2.1.102 2.1.103 2.1.104 2.1.105 2.1.200 2.1.201 2.1.202 2.1.300 2.1.301 2.1.302 2.1.400 2.1.401 2.1.402 2.1.403 2.1.500 2.1.502 2.1.503 2.1.504 2.1.505 2.2.100 2.2.101 2.2.102 2.2.103 2.2.104 2.2.105
-- Go 1.13.6
+- Go 1.13.7
 
 ### Package Management
+- Rustup 1.21.1
```
Comment (miketimofeev) [inline images/macos/macos-10.14-Readme.md]: Don't need in the description 😃

File: images/macos/macos-10.14-Readme.md [hunk 3]
```diff
@@ -4,34 +4,26 @@ The following software is installed on machines in the Azure Pipelines **macOS-1
 
 #### Xcode 11.3.1 set by default
 
-## OS X info
```
[the remark this answers is not in the document]
Comment (al-cheb) [reply images/macos/macos-10.14-Readme.md]: https://github.com/microsoft/azure-pipelines-image-generation/pull/1470#discussion_r370118262

[the remark this answers is not in the document]
Comment (miketimofeev) [reply images/macos/macos-10.14-Readme.md]: Yeah, but these lines still present in that PR
## Operating System
- OS X 10.13.6 (17G10021) **High Sierra**

**Pull request merged**